### PR TITLE
CCD-4056: Move CVE-2016-3094 to false positives

### PR DESCRIPTION
**Pull request by JamesCollettCGI**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CCD-4056

### Change description ###
Moved CVE-2016-3094 to false positives.   Affected dependency is org.apache.qpid/qpid-client (not org.apache.qpid/qpid-jms-client).
Removed CVE-2016-4432 as no longer issue.

**Does this PR introduce a breaking change?**
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
+  <!--Please add all the temporary suppressions under the below section-->
   <suppress>
     <notes>Temporary Suppression
-        CVE-2022-41946 refer [Ticket]
-        CVE-2016-4432 refer https://tools.hmcts.net/jira/browse/CCD-4056
-        CVE-2016-3094 refer https://tools.hmcts.net/jira/browse/CCD-4056
-        CVE-2016-4432 refer [Ticket]
-        CVE-2016-3094 refer [Ticket]
-        CVE-2021-37533 refer [Ticket]</notes>
+      CVE-2022-41946 refer [Ticket]
```
Comment (rajaram-nagarajan) [inline config/owasp/suppressions.xml:15]: @JamesCollettCGI - add the ticket details for these CVEs.

Comment (JamesCollettCGI) [reply config/owasp/suppressions.xml]: Fixed CVE-2022-41946 (postgresql version)   and   Added CVE-2021-37533 ticket ref.

Comment (rajaram-nagarajan) [reply config/owasp/suppressions.xml]: CVE-2022-41946 is already fixed and ready to be deployment.

Comment (rajaram-nagarajan) [reply config/owasp/suppressions.xml]: @JamesCollettCGI - can you resolve the conflicts?

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Pull request merged**